man/tr.man

```diff
@@ -0,0 +1,61 @@
+.TH VRMS-RPM 1
+.SH NAME
+vrms-rpm - Ücretsiz olmayan yüklü yazılımların rapor edilmesi
+.SH SYNOPSIS
+\fBvrms-rpm\fR \fI[OPTION]\fR ...
```

Comment:
The manpage uses `[seçenekler]` for options; I guess that should also be the case here.